### PR TITLE
Fix - fix support in any cnange deploy

### PR DESCRIPTION
**Pull request by GiliFaroEnv0**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
fix support in any cnange deploy
### Solution

we needed to send the `auto_deploy_on_path_changes_only` as `false` any way so added default

### CODE REVIEW

**Comment by GiliFaroEnv0** [comment]
[Environment resource doesn't support "run on any change" trigger](https://trello.com/c/9rQtigkZ/4966-environment-resource-doesnt-support-run-on-any-change-trigger)

**Review by eranelbaz — APPROVED** [review]

File: env0/resource_environment.go [hunk 1]
```diff
@@ -77,12 +77,14 @@ func resourceEnvironment() *schema.Resource {
 				Type:        schema.TypeBool,
 				Description: "redeploy only on path changes only",
 				Optional:    true,
+				Default:     false,
```
Comment (liranfarage89) [inline env0/resource_environment.go:80]: it's supposed to be true, but if the user set that to false and no glob pattern, then it should be anyChange no?
@GiliFaroEnv0

Comment (eranelbaz) [reply env0/resource_environment.go:80]: if we get to that we need to fetch the default from the BE

Comment (GiliFaroEnv0) [reply env0/resource_environment.go:80]: @liranfarage89 since we want the default behavior to be "on any change" this need to be false

Comment (liranfarage89) [reply env0/resource_environment.go:80]: on any change is not the default behavior as far as I know..

Comment (liranfarage89) [reply env0/resource_environment.go:80]: check the ui and see

Comment (liranfarage89) [reply env0/resource_environment.go:80]: @GiliFaroEnv0

**Pull request merged**